shell/components/fleet/dashboard/Empty.vue

```diff
@@ -0,0 +1,77 @@
+
+<script>
+import { FLEET } from '@shell/config/types';
+import { NAME } from '@shell/config/product/fleet';
+export default {
+  name: 'FleetDashboardEmpty',
+
+  props: {
+    permissions: {
+      type:     Object,
+      required: true
+    },
+  },
+
+  data() {
+    return {
+      getStartedLink: {
+        name:   'c-cluster-product-resource-create',
+        params: {
+          product:  NAME,
+          resource: FLEET.GIT_REPO
+        },
+      },
+    };
+  }
+};
+</script>
+<template>
+  <div
+    class="fleet-empty-dashboard"
+    :data-testid="'fleet-empty-dashboard'"
+  >
+    <i class="icon-fleet mb-30" />
```

Comment:
I think that we can add `aria-hidden="true"` to icons so that they won't be picked up be screen readers. 

FYI @aalves08